sysl2/sysl/integration/viewGen.go

```diff
@@ -0,0 +1,593 @@
+package integration
+
+import (
+	"fmt"
+	"sort"
+	"strconv"
+	"strings"
+
+	"github.com/anz-bank/sysl/src/proto"
+	"github.com/anz-bank/sysl/sysl2/sysl/utils"
+)
+
+type IntsParam struct {
+	apps         []string
+	highlights   map[string]struct{}
+	integrations []*AppDependency
+	app          *sysl.Application
+	endpt        *sysl.Endpoint
+}
+
+type Args struct {
+	title     string
+	project   string
+	clustered bool
+	epa       bool
+}
+
+type IntsDiagramVisitor struct {
+	m          *sysl.Module
+	sb         *strings.Builder
+	highlights map[string]struct{}
```

Comment:
@JackLeeHal give `m` and `sb` better names.

Reply:
changed to mod and stringBuilder